frontend/templates/product/sections/ServiceValuePropositionSection/index.tsx

```diff
@@ -53,40 +40,26 @@ export function ServiceValuePropositionSection() {
          >
             <ValueProposition>
                <ValuePropositionIcon icon={faBoxCircleCheck} />
-               <Title>Satified or 100% refunded</Title>
-               <Description>
+               <Text fontWeight="bold">Satified or 100% refunded</Text>
+               <Text>
```

Comment:
Why the change on all of these?

Reply:
We did also some checks with Mattia in this PR and we noticed that the fontSize was not correct for the description. As a consequence I took the chance to do some refactoring.
`Title` and `Description` component had become so simple that even creating those as separate components not necessary anymore